main.go

```diff
@@ -2,49 +2,37 @@ package main
 
 import (
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
 	"io/ioutil"
-	"log"
 	"net/http"
 	"os"
+	"path/filepath"
+	"strings"
 	"time"
 
-	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-steputils/stepconf"
+	"github.com/bitrise-io/go-utils/log"
 )
 
-const cacheArchivePath = "/tmp/cache-archive.tar"
-
-var (
-	gIsDebugMode = false
-)
-
-// StepParamsModel ...
-type StepParamsModel struct {
-	CacheAPIURL string
-	IsDebugMode bool
+// Config stores the step inputs.
+type Config struct {
+	CacheAPIURL string `env:"cache_api_url"`
```

Comment:
It should be a required input

Reply:
Or remove the required flag from the step.yml

Reply:
it is not required, step.yml modified: https://github.com/bitrise-steplib/steps-cache-pull/blob/master/main.go#L164